styles/fabricmc/catppuccin.user.css

```diff
@@ -0,0 +1,695 @@
+    --ctp-overlay2: @overlay2;
+    --ctp-overlay1: @overlay1;
+    --ctp-overlay0: @overlay0;
+    --ctp-surface2: @surface2;
+    --ctp-surface1: @surface1;
+    --ctp-surface0: @surface0;
+    --ctp-base: @base;
+    --ctp-mantle: @mantle;
+    --ctp-crust: @crust;
+
+    color-scheme: if(@lookup =latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    a:link,
+    a:visited,
+    a {
+      color: @accent-color;
+    }
+
+    body {
+      background-color: @base;
+      color: @text;
+    }
+
+    // <select> and <option> element styles are mostly used on fabricmc.net/develop
+    select,
+    input {
+      background-color: @mantle !important;
+      border-color: @surface1;
+    }
+
+    input[type="checkbox"] {
+      accent-color: @accent-color;
+    }
+
+    option:checked {
+      background-color: @crust;
+    }
+
+    pre,
+    code {
+      border-color: @surface1;
+      background-color: @mantle !important;
+    }
+
+    hr {
+      background-color: @surface1;
+    }
+
+    &:not(:has(#dokuwiki__site)) table {
+      border-color: @surface1;
+      tr:nth-child(even),
+      th {
+        background-color: @mantle;
+      }
+      td,
+      th {
+        border-color: @surface1;
+      }
+    }
+
+    // Website header
+    .site {
+      &-title,
+      &-title:visited,
+      &-nav .page-link {
+        color: @text !important;
+      }
+      // TOC Navbox for mobile
+      @media (max-width: 600px) {
+        &-nav {
+          background-color: @mantle;
+          border-color: @surface0;
+          .menu-icon > svg {
+            fill: @text;
+          }
+        }
+      }
+    }
+
+    button,
+    .button {
+      color: @text !important;
+    }
+
+    .primary {
+      background-color: @accent-color;
+      color: @crust !important;
+      &:hover,
+      &:focus {
+        background-color: fadeout(@accent-color, 10%);
+      }
+      &:active {
+        background-color: lighten(@accent-color, 5%);
+      }
+    }
+
+    .secondary {
+      border-color: @accent-color;
+      &:hover,
+      &:focus {
+        background-color: @accent-color;
+        color: @crust !important;
+        border-color: @accent-color;
+      }
+      &:active {
+        @active-accent: lighten(@accent-color, 10%);
+        background-color: @active-accent;
+        border-color: @active-accent;
+      }
+    }
+
+    .home .column .javadoc-selector select {
+      border-color: @accent-color;
+      box-shadow: @crust 1.95px 1.95px 2.6px;
+      option {
+        background-color: @mantle;
+      }
+    }
+
+    .post-meta {
+      color: @subtext0;
+    }
+
+    // Diff line removal
+    .gd {
+      background-color: fadeout(@red, 60%) !important;
+    }
+
+    // Diff line addition
+    .gi {
+      background-color: fadeout(@green, 60%) !important;
+    }
```

Comment:
Should be 20% opacity at most as per https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md#code-editors. Might want to group these lines with the other syntax highlighting as well.

Reply:
Also these two need to have `color` set (to text or whatever), the default website styles reset it.

Reply:
> Might want to group these lines with the other syntax highlighting as well.
These are for the blog component of main website and are not used on the wiki (where the GeSHi ruleset is). I did change it so it is grouped them with other styles relating to the blogs syntax highlighting however.